### DISCUSSION TITLE
How can I get the deformation of a structure in PyDPF..?

### DISCUSSION THREAD

**Discussion by Naveen-Kumar-B** [ansys/pydpf-post#583] [Q&A]
I can be able to plot the displacement of a structure, but the deformation is not showing. Can I get solution on this?

**Comment by PProfizi** [comment]
Hi @B-Naveen-Kumar,
Supposing the deformation result is available in your result file, you should be able to use the same method as for `displacement`, only this time it is called `strain`, or a name containing `strain`.
Please share a snippet of your code if you need more help.

**Reply by Naveen-Kumar-B** [reply]
Hi @PProfizi 
I have shared the code. 
![1](https://github.com/ansys/pydpf-post/assets/158252511/9af195c0-79e9-468f-aee1-ce5ef93d353d)
Output:
![crankshaft_stress_xx](https://github.com/ansys/pydpf-post/assets/158252511/0ab2fa79-da5b-4d74-b05b-a0c6cafcd597)

**Reply by PProfizi** [reply]
Hi @B-Naveen-Kumar,

Ah yes, sorry, I misunderstood your initial question.
I just checked and right now the option to plot a `DataFrame` with deformation does not exist.
I am starting an issue to add this option to the `DataFrame.plot` method.
It should not be too long to do.
Thank you for letting me know!

**Reply by PProfizi** [reply]
https://github.com/ansys/pydpf-post/issues/586

**Reply by Naveen-Kumar-B** [reply]
Hi @PProfizi 
Many thanks for raising the issue.
